controllers/ManagementController.php

```diff
@@ -195,7 +196,9 @@ private function createAccount(CreateAccountForm $form)
         $user->is_active = true;
         $user->role         = $form->role;
 
-        if(!$user->save()){
+        if($user->save()){
```

Comment:
You don't need the if else structure, just make ```if(!$user->save()) {
throw new CannotSaveException($user) 
}```
and then you redirect to the account newly created

Reply:
I agree !